stdlib/ballerina-http/src/main/ballerina/http/ws/websocket_connector.bal

```diff
@@ -38,46 +38,45 @@ public type WebSocketConnector object {
         P{{final}} True if this is a final frame of a (long) message
         R{{}} `error` if an error occurs when sending
     }
-    public native function pushBinary(blob data, boolean final = true) returns error|();
+    public native function pushBinary(blob data, boolean final = true) returns error?;
 
     documentation {
         Ping the connection.
 
         P{{data}} Binary data to be sent.
         R{{}} `error` if an error occurs when sending
     }
-    public native function ping(blob data) returns error|();
+    public native function ping(blob data) returns error?;
 
     documentation {
         Send pong message to the connection.
 
         P{{data}} Binary data to be sent
         R{{}} `error` if an error occurs when sending
     }
-    public native function pong(blob data) returns error|();
+    public native function pong(blob data) returns error?;
 
     documentation {
         Close the connection.
 
         P{{statusCode}} Status code for closing the connection
         P{{reason}} Reason for closing the connection
-        P{{timeoutInSecs}} timeout which waits for the close frame from remote endpoint to close the connection.
-                           If the timeout exceeds then the connection will be terminated even though a close frame
-                           is not received from the remote backend. If the value is -1 the connection will wait until
-                           a close frame is received.
-                           Default value 0: After sending the close frame connection will be closed automatically
-                           (Timeout is 0).
+        P{{timeoutInSecs}} Timeout which waits for the close frame from remote endpoint before closing the connection.
```

Comment:
Timeout which waits for -> Time to wait for 
from remote endpoint -> from the remote endpoint